tests/baselines_test/dads_smerl_test.py

```diff
@@ -31,7 +31,8 @@ def test_dads_smerl() -> None:
     tau = 0.005
     grad_updates_per_step = 0.25
     normalize_observations = False
-    hidden_layer_sizes = (256, 256)
+    critic_hidden_layer_size: tuple = (256, 256)
+    policy_hidden_layer_size: tuple = (64, 64)
```

Comment:
we should probably keep the same default behavior, hence use (256, 256) as default here. Don't you think?